packages/next/src/build/webpack/plugins/telemetry-plugin/telemetry-plugin.ts

```diff
@@ -127,16 +119,16 @@ const useCacheTracker = createUseCacheTracker()
 /**
  * Determine if there is a feature of interest in the specified 'module'.
  */
-function findFeatureInModule(module: Module): Feature | undefined {
+function findFeatureInModule(module: webpack.Module): Feature | undefined {
   if (module.type !== 'javascript/auto') {
     return
   }
-  const normalizedIdentifier = module.identifier().replace(/\\/g, '/')
-  for (const [feature, path] of FEATURE_MODULE_MAP) {
-    if (normalizedIdentifier.endsWith(path)) {
+  for (const [feature, rawRequest] of FEATURE_MODULE_MAP) {
+    if ((module as webpack.NormalModule).rawRequest === rawRequest) {
```

Comment:
qq: if a library using import xx from `'next/image.js'` due to ESM module requirement, will this get effected?

Reply:
Good point. Let me check if there's a better field that gives me the resolved module.